controllers/ytsaurus_controller_test.go

```diff
@@ -173,73 +173,20 @@ type testRow struct {
 
 var _ = Describe("Basic test for Ytsaurus controller", func() {
 	Context("When setting up the test environment", func() {
-		It("Should run and update Ytsaurus", func() {
-
-			By("Creating a Ytsaurus resource")
-			ctx := context.Background()
-
-			namespace := "test1"
-
-			ytsaurus := ytv1.CreateBaseYtsaurusResource(namespace)
-
-			g := ytconfig.NewGenerator(ytsaurus, "local")
-
-			defer deleteYtsaurus(ctx, ytsaurus)
-			runYtsaurus(ytsaurus)
-
-			By("Creating ytsaurus client")
-
-			ytClient := getYtClient(g, namespace)
-
-			By("Check that cluster alive")
-
-			res := make([]string, 0)
-			Expect(ytClient.ListNode(ctx, ypath.Path("/"), &res, nil)).Should(Succeed())
-
-			By("Check that tablet cell bundles are in `good` health")
-
-			Eventually(func() bool {
-				notGoodBundles, err := components.GetNotGoodTabletCellBundles(ctx, ytClient)
-				if err != nil {
-					return false
-				}
-				return len(notGoodBundles) == 0
-			}, timeout, interval).Should(BeTrue())
-
-			By("Run cluster update")
-
-			Expect(k8sClient.Get(ctx, types.NamespacedName{Name: ytv1.YtsaurusName, Namespace: namespace}, ytsaurus)).Should(Succeed())
-			ytsaurus.Spec.CoreImage = ytv1.CoreImageSecond
-			Expect(k8sClient.Update(ctx, ytsaurus)).Should(Succeed())
-
-			Eventually(func() bool {
-				ytsaurus := &ytv1.Ytsaurus{}
-				err := k8sClient.Get(ctx, types.NamespacedName{Name: ytv1.YtsaurusName, Namespace: namespace}, ytsaurus)
-				if err != nil {
-					return false
-				}
-				return ytsaurus.Status.State == ytv1.ClusterStateUpdating
-			}, timeout, interval).Should(BeTrue())
-
-			Eventually(func() bool {
-				ytsaurus := &ytv1.Ytsaurus{}
-				err := k8sClient.Get(ctx, types.NamespacedName{Name: ytv1.YtsaurusName, Namespace: namespace}, ytsaurus)
-				if err != nil {
-					return false
-				}
-				return ytsaurus.Status.State == ytv1.ClusterStateRunning
-			}, timeout*5, interval).Should(BeTrue())
-
-			By("Check that cluster alive after update")
-
-			Expect(ytClient.ListNode(ctx, ypath.Path("/"), &res, nil)).Should(Succeed())
-		})
+		It(
+			"Should run and update Ytsaurus to the same minor version",
```

Comment:
major

Reply:
In terms of YT all numbered versions are historically called "major" releases, and we use the term minor for updates within a single major version, i.e. from some 23.1 commit to a later 23.1 commit.

Reply:
So, if I've correctly understood from your comment, 23.1 and 23.2 are called major version and it would be correct to write something like:
`Should run and update Ytsaurus within same major version` for 23.1>23.1 case and 
`Should run and update Ytsaurus to the next major version` for 23.1>23.2 case.
?

Reply:
Fixed